tests/mtest/src/main/scala/tests/BaseSuite.scala

```diff
@@ -149,6 +149,11 @@ class BaseSuite extends TestSuite {
       myTests += FlatTest(name, () => fun)
     }
   }
+
+  def failingTest(name: String)(fun: => Any): Unit = {
+    test(name)(intercept[TestFailedException](fun))
```

Comment:
I'm tempted to do

```scala
    test(utest.ufansi.Color.LightRed(s"EXPECTED FAILURE - $name").toString()) {
      intercept[TestFailedException](fun)
    }
```

and obtain

![image](https://user-images.githubusercontent.com/691940/71780886-51c43800-2fc8-11ea-984c-16a32f316c65.png)


but then it becomes hard to select a single test to run.

Reply:
I agree it’s better to leave the test name unchanged.